modules/ossm-install-cert-manager.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-install-cert-manager_{context}"]
+== Install cert-manager
+
+.Procedure
+
+. Create root cluster issuer:
++
+[source, terminal]
+----
+oc apply -f cluster-issuer.yaml
+----
++
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-root-issuer
+  namespace: openshift-operators
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: root-ca
+  namespace: openshift-operators
+spec:
+  isCA: true
+  duration: 21600h # 900d
+  secretName: root-ca
+  commonName: root-ca.my-company.net
+  subject:
+    organizations:
+    - my-company.net
+  issuerRef:
+    name: selfsigned-root-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: root-ca
+spec:
+  ca:
+    secretName: root-ca
+----
++
+====
+[NOTE]
+The namespace of `selfsigned-root-issuer` issuer and `root-ca` certificate is `openshift-operators`. This is necessary because `root-ca` is a cluster issuer, so cert-manager will look for a referenced secret in its own namespace, which is `openshift-operators` in this case.
```

Comment:
```suggestion
The namespace of `selfsigned-root-issuer` issuer and `root-ca` certificate is `cert-manager`. This is necessary because `root-ca` is a cluster issuer, so cert-manager will look for a referenced secret in its own namespace, which is `cert-manager` in case of Red Hat cert-manager.
```

Reply:
Is "of Red Hat cert-manager" referring to "cert-manager Operator for Red Hat OpenShift"?